drivers/aufs/aufs_test.go

```diff
@@ -312,16 +317,16 @@ func TestCreateWithInvalidParent(t *testing.T) {
 	d := newDriver(t)
 	defer os.RemoveAll(tmp)
 
-	if err := d.Create("1", "storage", "", nil); err == nil {
-		t.Fatalf("Error should not be nil with parent does not exist")
+	if err := d.Create("1", "docker", nil); err == nil {
```

Comment:
Lost a rename here?  It's test code, though.

Reply:
Fixed